src/sockperf.cpp

```diff
@@ -3114,6 +3107,13 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
             continue;
         }
 
+        if (tmp->server_addr.ss_family == AF_UNIX) {
+            s_user_params.tcp_nodelay = false;
+            strcpy(s_user_params.addr.addr_un.sun_path,tmp->server_addr.addr_un.sun_path);
```

Comment:
It's hard to me to understand where we perform `sun_path` length validation. Is string length in `tmp` already validated?

Reply:
`resolve_sockaddr` func validate if the length is more than MAX_UDS_NAME